frontend/src/bundles/common/components/transaction-modal/transaction-modal.tsx

```diff
@@ -108,20 +108,22 @@ const TransactionModal: React.FC<Properties> = ({
             }
             submitButtonName={submitButtonName}
         >
-            <TransactionImage
-                file={imageFile}
-                handleFileChange={handleFileChange}
-            />
-            {type === TransactionModalType.CHANGE && (
-                <Button
-                    className={styles.delete}
-                    type={ButtonType.BUTTON}
-                    size={ButtonSize.SMALL}
-                    variant={ButtonVariant.SECONDARY}
-                >
-                    Delete Transaction
-                </Button>
-            )}
+            <div className={styles.buttons_container}>
```

Comment:
`className={styles.buttonContainer}` we use camelCase in class naming

Reply:
changed